api/src/opentrons/protocol_api/labware.py

```diff
@@ -395,10 +395,28 @@ def quirks(self) -> List[str]:
         """Quirks specific to this labware."""
         return self._core.get_quirks()
 
-    # TODO(mc, 2022-09-23): use `self._core.get_default_magnet_engage_height`
+    # TODO(mm, 2023-02-08):
+    # Specify units and origin after we resolve RSS-110.
+    # Remove warning once we resolve RSS-109 more broadly.
     @property  # type: ignore
     @requires_version(2, 0)
     def magdeck_engage_height(self) -> Optional[float]:
+        """Return the default magnet engage height that
+        :py:meth:`MagneticModuleContext.engage` will use for this labware.
+
+        .. warning::
+            This currently returns confusing and unpredictable results that do not
+            necessarily match what :py:meth:`MagneticModuleContext.engage` will
```

Comment:
```suggestion
        :py:meth:`.MagneticModuleContext.engage` will use for this labware.

        .. warning::
            This currently returns confusing and unpredictable results that do not
            necessarily match what :py:meth:`.MagneticModuleContext.engage` will
```
doesn't link properly without the leading dot

Reply:
Nice catch. Serves me right for trying to write ReST syntax from memory.